### DISCUSSION TITLE
Certificate for api.quotable.io expired September 10, 2024

### DISCUSSION THREAD

**Discussion by neilfairall** [lukePeavey/quotable#240] [Report Downtime]
As stated in title: Certificate for api.quotable.io expired September 10, 2024

**Comment by neilfairall** [comment]
The (obviously undesirable) workaround is to skip certificate checking for the time being, as in

```
function Get-Quote {
    <#
    
        Get a quote from quotable.io, and parse it so that you get it formatted nicely

    #>
    $url = "https://api.quotable.io/quotes/random?limit=1"
    $tmp = (Invoke-WebRequest -uri $url -SkipCertificateCheck)
    $tmp = [regex]::Split( $tmp.content , ',(?=(?:[^"]|"[^"]*")*$)' )
    (($tmp -match '"content"' -split ":") -replace '"','')[1]
    ' - '+(($tmp -match '"author"' -split ":") -replace '"','')[1]
    }
```

**Reply by Codeusername78** [reply]
Its been quite some time, but is this safe??

**Comment by conceptia2020** [comment]
Does anyone know what the root cause(s) of the issues with quotable.io are? I'm no longer a developer, but I do know that there are reasons why things keep happening...especially when you have a tool like this that does one thing and does it well. I'd like to contribute somehow (even financially) to help get quotable and stay up but have no idea what's going on. Does anyone know Luke Peavey? Or does anyone have an idea of the main 1 or 2 issues that keep taking quotable.io down?

**Comment by MonicaSue** [comment]
Any update on the expired certificate?

**Comment by Universe-Man** [comment]
Chiming in that I am experiencing the SSL cert issue on my requests still as of today.

**Comment by diskostu** [comment]
Yep, not fixed until today.

**Comment by conceptia2020** [comment]
Does anyone know what it will actually take to get this running again? Like, if I paid for a cert could it be used for this somehow? (Also, if this is a dumb question, can someone let me know what a better question would be?)
